### PR TITLE
Revise section "Record arguments"

### PR DESCRIPTION
**Pull request by jugmac00**
Revise section "Record Arguments"
    
- improve formatting
- fix typos
- lowercase all html tags
    
modified:   docs/zdgbook/ObjectPublishing.rst

### CODE REVIEW

File: docs/zdgbook/ObjectPublishing.rst [hunk 1]
```diff
@@ -826,103 +826,101 @@ Method Arguments
 ----------------
 
 Sometimes you may wish to control which object is published based on
-form data. For example, you might want to have a form with a select
```
Comment (dataflake) [inline docs/zdgbook/ObjectPublishing.rst]: the old line was correct

File: docs/zdgbook/ObjectPublishing.rst [hunk 2]
```diff
@@ -826,103 +826,101 @@ Method Arguments
...
+through the use of the ``method`` argument type. The method type allows
+the request variable ``PATH_INFO`` to be augmented using information
+from a form item's name or value.
+
+If the name of a form field is ``:method``, then the value of the field
+is added to ``PATH_INFO``. For example, if the original ``PATH_INFO``
+is ``foo/bar`` and the value of a ``:method`` field is ``x/y``, then
+``PATH_INFO`` is transformed to ``foo/bar/x/y``. This is useful when
 presenting a select list. Method names can be placed in the select
-option values.
```
Comment (dataflake) [inline docs/zdgbook/ObjectPublishing.rst]: old line is correct

File: docs/zdgbook/ObjectPublishing.rst [hunk 3]
```diff
@@ -826,103 +826,101 @@ Method Arguments
...
-therefore, not contain method names.
+If the name of a form field **ends** in ``:method`` then the part of
+the name before ``:method`` is added to ``PATH_INFO``. For example, if
+the original ``PATH_INFO`` is ``foo/bar`` and there is a ``x/y:method``
+field, then ``PATH_INFO`` is transformed to ``foo/bar/x/y``. In this
+case, the form value is ignored. This is useful for mapping submit
+buttons to methods, since submit button values are displayed and
+should therefore not contain method names.
 
-Only one method field should be provided. If more than one method
```
Comment (dataflake) [inline docs/zdgbook/ObjectPublishing.rst]: why did you remove this?

File: docs/zdgbook/ObjectPublishing.rst (hunk 1 above)
Comment (jugmac00) [reply docs/zdgbook/ObjectPublishing.rst]: Sorry - pulled the review button to early - this change was done in the other pull request - and fixed there already.

File: docs/zdgbook/ObjectPublishing.rst (hunk 2 above)
Comment (jugmac00) [reply docs/zdgbook/ObjectPublishing.rst]: Sorry pushed the review button to early. This change has been done in the other pull request - and already reverted there.

File: docs/zdgbook/ObjectPublishing.rst (hunk 3 above)
Comment (jugmac00) [reply docs/zdgbook/ObjectPublishing.rst]: Sorry, this change belongs also to the other pull request - and there I put the explanation in the commit message as follows:
>    deleted note about not to use more than one method field, as
>    it is a common use case to create forms with several input buttons,
>    each with a method argument

**Comment by jugmac00** [comment]
@dataflake I am very sorry - by mistake I included the commit for #566 also in this pr (now I force pushed this branch again - without the duplicate commit).
